low-level/framework/rabbitmq/rabbitmq_egress_accumulated_msgs_processor.py

```diff
@@ -117,11 +106,10 @@ def run(self):
                                     "global shutdown message from sspl_ll_d")
                     self.shutdown()
         try:
+            # TODO : Fix accumulated message processor when message bus changes are available to 
```

Comment:
create a jira ticket (7 mention here) for this sspl change, depending on message bus ticket, so we dont miss out

Reply:
line is too long.